src/test/java/org/jenkinsci/plugins/android_lint/LintWorkflowTest.java

```diff
@@ -0,0 +1,77 @@
+package org.jenkinsci.plugins.android_lint;
+
+import hudson.FilePath;
+import hudson.model.Result;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertTrue;
+
+public class LintWorkflowTest {
+
+    @Rule
+    public JenkinsRule jenkinsRule = new JenkinsRule();
+
+    /**
+     * Run a workflow job using {@link LintPublisher} and check for success.
+     */
+    @Test
+    public void lintPublisherWorkflowStep() throws Exception {
+        WorkflowJob job = jenkinsRule.jenkins.createProject(WorkflowJob.class, "lintPublisherWorkflowStep");
+        FilePath workspace = jenkinsRule.jenkins.getWorkspaceFor(job);
+        FilePath report = workspace.child("target").child("lint-results.xml");
+        report.copyFrom(LintWorkflowTest.class.getResourceAsStream("/org/jenkinsci/plugins/android_lint/parser/lint-results_r20.xml"));

```

Comment:
This would be simpler as `report.copyFrom(getClass().getResourceAsStream("parser/lint-results_r20.xml"))`?